composer/workflows/bq_copy_across_locations.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example Airflow DAG that performs an export from BQ tables listed in
+config file to GCS,
+copies GCS objects across locations (e.g., from US to EU) then imports from
+GCS to BQ. The DAG
+imports the gcs_to_gcs operator from plugins and dynamically builds the
+tasks based on the
+list of tables.
+Lastly, the DAG defines a specific application logger to generate logs.
+
+This DAG relies on three Airflow variables
+https://airflow.apache.org/concepts.html#variables
+* master_file_path - CSV file listing source and target tables, including
+Datasets.
+* gcs_source_bucket - Google Cloud Storage bucket to use for exporting
+BigQuery tables in source.
+* gcs_dest_bucket - Google Cloud Storage bucket to use for importing
+BigQuery tables in
+                    destination.
+  See https://cloud.google.com/storage/docs/creating-buckets for creating a
+  bucket.
+"""
+
+# --------------------------------------------------------------------------------
+# Load The Dependencies
+# --------------------------------------------------------------------------------
+
+import csv
+from datetime import datetime
+from datetime import timedelta
+import logging
+
+from airflow import models
+from airflow.contrib.operators.bigquery_to_gcs import \
```

Comment:
Import modules not classes. `from airflow.contrib.operators import bigquery_to_gcs`. Likewise for other imports.

Reply:
Done